pkg/scheduler/apis/extender/v1/api.go

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
I would name the file types.go, api is not useful because it is already under ```scheduler/apis/```

Reply:
`types.go` is somehow the naming conversion of Kubernetes' API Machinery which just make me think of API Machinery's type system as well as client-gen, conversion-gen, etc.

But here, this file is more like `pkg/kubelet/apis/deveiceplugin/<version>/api.go`, so I make the name `api.go` here, to intentionally differentiate it with `types.go` as we don't have scheme, conversion logic here.

Reply:
`types.go` is somehow the naming conversion of Kubernetes' API Machinery which just make me think of API Machinery's type system as well as client-gen, conversion-gen, etc.

But here, this file is more like `pkg/kubelet/apis/deveiceplugin/<version>/api.go`, so I made the name `api.go` here, to intentionally differentiate it with types.go as we don't have scheme, conversion logic here.

Reply:
sg

Reply:
well.. I changed my mind.. I think renaming to `types.go` _probably_ makes a bit more sense as we leveraged the "deepcopy codegen" here, will rename to `types.go`.